modules/random-list/random-list.nu

```diff
@@ -92,7 +92,7 @@ export def "random-list integer" [
     }
 
     1..$list_length | each {|it|
-        random integer $range
+        random int $range
```

Comment:
Should we rename the `random-list integer` function to `random-list int` as well?

Reply:
yeeeeeah maybe, good catch :+1:

Reply:
done in 4b230c9

Reply:
Yo dawg, I heard you like dealing with breaking changes so I put a breaking change into you dealing with breaking changes.

Reply:
:laughing:

Reply:
i guess it's fine to not deprecate this in the `nu_scripts`